packages/core/src/classes/Abstracts/__tests__/Config.abstract.class.test.ts

```diff
@@ -70,4 +70,10 @@ describe("Config.buildArgs()", () => {
       );
     }
   });
+
+  it("should set the referral fee", () => {
+    const config = new TestClass();
+    config.setSkipReferral(true);
+    expect(config.skipReferral).toBe(true);
+  });
```

Comment:
The test description isn't correct, is it? It looks to me as the test is about _skipping_ the referral fee instead of actually setting it.